src/Algebra/Graph/AdjacencyIntMap/Algorithm.hs

```diff
@@ -117,20 +118,22 @@ bfsForestFrom' vs g = evalState (explore vs) IntSet.empty where
 -- @
 -- bfs vs 'empty'                                         == []
 -- bfs [] g                                             == []
--- bfs [1] ('edge' 1 1)                                   == [[1]]
--- bfs [1] ('edge' 1 2)                                   == [[1],[2]]
--- bfs [2] ('edge' 1 2)                                   == [[2]]
+-- bfs [1] ('edge' 1 1)                                   == [[[1]]]
+-- bfs [1] ('edge' 1 2)                                   == [[[1],[2]]]
+-- bfs [2] ('edge' 1 2)                                   == [[[2]]]
 -- bfs [3] ('edge' 1 2)                                   == []
--- bfs [1,2] ('edge' 1 2)                                 == [[1],[2]]
--- bfs [2,1] ('edge' 1 2)                                 == [[2],[1]]
+-- bfs [1,2] ('edge' 1 2)                                 == [[[1],[2]]]
+-- bfs [2,1] ('edge' 1 2)                                 == [[2]],[[1]]
```

Comment:
Aha, this is exactly my example from #233 :) Different number of "components" depending on the order of source vertices.